packages/common/src/utils/filters.ts

```diff
@@ -139,6 +135,8 @@ export const getFilterTypeFromItem = (item: FilterableItem): FilterType => {
             return FilterType.BOOLEAN;
         default: {
             return assertUnreachable(
+                // @ts-expect-error - TODO: Fix this
+                // TODO: Fix this
```

Comment:
❓ Need help with this - this appeared after changes to `FilterableItem` type

Reply:
Do we need to handle custom dimensions? I was hoping to do this in the other ticket.

```
const type = isCustomBinDimension ? DimensionType.STRING : isCustomSqlDimension ? item.dimensionType : item.type;
switch (type)
...
```

Reply:
Nope, we dont... `FilterableField` does not contain `CustomDimension`s...

Reply:
https://github.com/lightdash/lightdash/pull/10062/commits/ec23c3cb8dd1acde51d62ea9d3f4ecf62a6e4a36

This removes the reference of `itemsMap` in `FilterableField` because that was include `Field`, and field as a `type: string` (**not an enum**) - hence why that `assertUnreachable` was complaining about a `string`